Utilities.php

```diff
@@ -73,29 +73,31 @@ public function buildTree($route, $mode = false, $depth = 0)
         $pages = $page->evaluate([$mode => $route]);
         $pages = $pages->published()->order($config['order']['by'], $config['order']['dir']);
         $paths = array();
-        foreach ($pages as $page) {
-            if ($config['exclude_modular'] && isset($page->header()->content['items'])) {
-                if ($page->header()->content['items'] == '@self.modular') {
-                    continue;
-                }
-            }
-            $route = $page->rawRoute();
-            $path = $page->path();
-            $title = $page->title();
-            $paths[$route]['depth'] = $depth;
-            $paths[$route]['title'] = $title;
-            $paths[$route]['route'] = $route;
-            $paths[$route]['name'] = $page->name();
-            if (!empty($paths[$route])) {
-                $children = $this->buildTree($route, $mode, $depth);
-                if (!empty($children)) {
-                    $paths[$route]['children'] = $children;
-                }
-            }
-            $media = new Media($path);
-            foreach ($media->all() as $filename => $file) {
-                $paths[$route]['media'][$filename] = $file->items()['type'];
-            }
+        if ($depth <= $config['max_depth']) {
```

Comment:
This should work fine, but ensure the indentation matches the four spaces used elsewhere.

Reply:
Corrected.